index.js

```diff
@@ -5,6 +5,7 @@ module.exports = app => {
   app.on(['pull_request.opened', 'pull_request.synchronize', 'check_run.rerequested'], check)
 
```

Comment:
```suggestion
  const timeStart = new Date()
```

Reply:
hmm This actually needs to happen before line 5 but it wouldn't let me suggest that 😓